matRad_calcParticleDoseMC.m

```diff
@@ -202,13 +207,16 @@
     % allocate empty target point container
     for j = 1:numel(stfMCsquare(i).energies)
         stfMCsquare(i).energyLayer(j).targetPoints = [];
+        stfMCsquare(i).energyLayer(j).numOfPrimaries = [];
     end
     
     for j = 1:stf(i).numOfRays
         for k = 1:numel(stfMCsquare(i).energies)
             if any(stf(i).ray(j).energy == stfMCsquare(i).energies(k))
                 stfMCsquare(i).energyLayer(k).targetPoints = [stfMCsquare(i).energyLayer(k).targetPoints; ...
                                         -stf(i).ray(j).rayPos_bev(1) stf(i).ray(j).rayPos_bev(3)];
+                stfMCsquare(i).energyLayer(k).numOfPrimaries = [stfMCsquare(i).energyLayer(k).numOfPrimaries ...
+                                         stf(i).ray(j).weight(stf(i).ray(j).energy == stfMCsquare(i).energies(k))];
```

Comment:
This line crashes when doing a dij calculation. stf(i).ray(j).weight does not seem to exist there.

Reply:
Jap. I forgot to push the fix for this yesterday. This line should only be executed for calcDoseDirect option. If not then we need to write a dummy value... I will push a fix later.